<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlayerWindow</class>
 <widget class="QMainWindow" name="PlayerWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>972</width>
    <height>493</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>My Video Player</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="VideoRenderWidget" name="videoWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>640</width>
        <height>360</height>
       </size>
      </property>
      <property name="styleSheet">
       <string>background-color: black;</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="progressLayout">
      <item>
       <widget class="QLabel" name="labelCurrentTime">
        <property name="text">
         <string>00:00</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="ProgressSlider" name="sliderProgress">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelTotalTime">
        <property name="text">
         <string>/ 00:00</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="controlLayout">
      <item>
       <widget class="QPushButton" name="btnOpen">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>üìÇ ÊâìÂºÄ</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnPlay">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>‚ñ∂Ô∏è Êí≠Êîæ</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnPause">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>‚è∏ ÊöÇÂÅú</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnStop">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>‚èπ ÂÅúÊ≠¢</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnRewind">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>‚è™ Âø´ÈÄÄ</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnForward">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>‚è© Âø´Ëøõ</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnScreenshot">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>üì∏ Êà™Âõæ</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="spacerRight">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Policy::Expanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QComboBox" name="comboSpeed">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>30</height>
         </size>
        </property>
        <item>
         <property name="text">
          <string>0.5x</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>0.75x</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>1.0x</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>1.5x</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2.0x</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>3.0x</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnVolumeIcon">
        <property name="minimumSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>üîä</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="ClickableSlider" name="sliderVolume">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="value">
         <number>50</number>
        </property>
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelVolumeValue">
        <property name="minimumSize">
         <size>
          <width>30</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>20</height>
         </size>
        </property>
        <property name="text">
         <string>50</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnFullscreen">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>üî≥ ÂÖ®Â±è</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>972</width>
     <height>33</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Êñá‰ª∂(&amp;F)</string>
    </property>
    <addaction name="actionOpenFile"/>
    <addaction name="actionOpenStream"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>ËßÜÂõæ(&amp;V)</string>
    </property>
    <addaction name="actionFullscreen"/>
    <addaction name="actionZoomMode"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Â∑•ÂÖ∑(&amp;T)</string>
    </property>
    <addaction name="actionScreenshot"/>
    <addaction name="actionVideoInfo"/>
    <addaction name="actionSettings"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Â∏ÆÂä©(&amp;H)</string>
    </property>
    <addaction name="actionShortcuts"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
   <addaction name="menuTools"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpenFile">
   <property name="text">
    <string>ÊâìÂºÄÊú¨Âú∞Êñá‰ª∂</string>
   </property>
  </action>
  <action name="actionOpenStream">
   <property name="text">
    <string>ÊâìÂºÄÁΩëÁªúÊµÅ</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>ÈÄÄÂá∫</string>
   </property>
  </action>
  <action name="actionFullscreen">
   <property name="text">
    <string>ÂÖ®Â±è</string>
   </property>
  </action>
  <action name="actionZoomMode">
   <property name="text">
    <string>Áº©ÊîæÊ®°Âºè</string>
   </property>
  </action>
  <action name="actionScreenshot">
   <property name="text">
    <string>Êà™Âõæ</string>
   </property>
  </action>
  <action name="actionVideoInfo">
   <property name="text">
    <string>ËßÜÈ¢ë‰ø°ÊÅØ</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>ËÆæÁΩÆ</string>
   </property>
  </action>
  <action name="actionShortcuts">
   <property name="text">
    <string>Âø´Êç∑ÈîÆËØ¥Êòé</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>ÂÖ≥‰∫é</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ClickableSlider</class>
   <extends>QSlider</extends>
   <header>ClickableSlider.h</header>
  </customwidget>
  <customwidget>
   <class>VideoRenderWidget</class>
   <extends>QOpenGLWidget</extends>
   <header>VideoRenderWidget.h</header>
  </customwidget>
  <customwidget>
   <class>ProgressSlider</class>
   <extends>QSlider</extends>
   <header>ProgressSlider.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
